mobile/components/theme/index.js

```diff
@@ -66,6 +66,7 @@ export default {
     alt: '#828C99',
     reverse: '#FFFFFF',
     placeholder: '#A3AFBF',
+    secondary: '#494C57',
```

Comment:
Let's also add this to the web theme.

Reply:
It already exists in the web theme. And yes, we should probably just share this file :)